src/components/ShoppingCart.vue

```diff
@@ -54,14 +54,12 @@ export default {
   components: { ShoppingCartTotalPrice },
   data() {
     return {
-      products: this.$shoppingCart.state.products
+      shoppingCart: this.$shoppingCart.state
```

Comment:
Pas possible d'utiliser directement `this.$shoppingCart.state.products`, sinon, `products` n'est pas observé par Vue (lorsque products change, le composant vue n'est pas modifié).